src/ray/common/cgroup/cgroup_context.h

```diff
@@ -0,0 +1,41 @@
+// Copyright 2024 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <unistd.h>
+
+#include <cstdint>
+#include <string>
+#include <string_view>
+
+namespace ray {
+
+// Context used to setup cgroupv2 for a task / actor.
+struct PhysicalModeExecutionContext {
+  // Directory for cgroup, which is appled to application process.
+  std::string_view cgroup_directory;
+  // UUID to indicate the current task / actor.
+  std::string uuid;
```

Comment:
[idea] can we use TaskID ?

Reply:
Yeah I plan to use task id, but we could also have actor id, which are two different concepts.

I'm thinking just to use the term `id`.

Reply:
Actor is not a problem since we can just use the creation task Id. However we are associated with task attempts not tasks so I suspect we can face some uniqueness problems. Maybe uuid is the way to go to the end. WDYT